authorization/v1/generated.pb.go

```diff
@@ -1053,7 +1053,7 @@ func init() {
 }
 
 var fileDescriptor_39b89822f939ca46 = []byte{
-	// 1821 bytes of a gzipped FileDescriptorProto
+	// 1820 bytes of a gzipped FileDescriptorProto
```

Comment:
Is there any reason there are changes to this file?  Looks like all changes to this file could be dropped (and then maybe the verify job would pass).

Reply:
hm. not sure why that changed. it wasn't intentional. i'm assuming it's okay for me to overwrite this even though the scripts changed it

Reply:
can someone verify, though? since this is auto-generated and the internal byte representation changed as well, i feel lke i should leave it as-is

Reply:
reverting the changes get me past the verify tests